core/gather/gatherers/bf-cache-errors.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import FRGatherer from '../base-gatherer.js';
+import {waitForFrameNavigated, waitForLoadEvent} from '../driver/wait-for-condition.js';
+
+class BFCacheErrors extends FRGatherer {
```

Comment:
maybe name to just `BFCache` or `BFCacheReasonsNotRestored`?

Reply:
I wanted to differentiate from the `BFCache` audit, and `BFCacheReasonsNotRestored` feels too long. This also mirrors the `InstallabilityErrors` artifact.